src/Facebook/InstantArticles/Elements/InstantArticle.php

```diff
@@ -396,6 +396,7 @@ public function addMetaProperty($property_name, $property_content)
 
     public function render($doctype = '<!doctype html>', $format = false)
     {
+        $doctype = is_null($doctype) ? '<!doctype html>' : $doctype;
```

Comment:
Thats true, and expected.
If needed to override that default parameter, you gotta inform `''` (empty string/quotes)